doremus.ttl

```diff
@@ -1601,6 +1601,40 @@ Examples:"""@en ;
                        rdfs:label "M9 Derivation Type"@en ,
                                   "M9 Type de dérivation"@fr .
 
+###  http://data.doremus.org/ontology#U83_has_relative_key
+mus:U83_has_relative_key rdf:type owl:SymmetricObjectProperty ;
+    rdfs:domain mus:M4_Key ;
+    rdfs:range mus:M4_Key ;
+    rdfs:isDefinedBy <http://data.doremus.org/ontology#> ;
+    rdfs:label "U83 has relative key"@en , "U83 a pour tonalité relative"@fr ;
+    rdfs:comment "Scope Note: This class allows to connect two M4 Keys that are in a relative relationship, i.e. one is the relative major or minor of the other one."@en ,
+    "Scope Note: Cette classe permet de connecter deux M4 Tonalité qui sont en relation par rapport à l'armure, c'est-à-dire l'une est la relatif majeur ou mineur de l'autre."@fr ,
+    "Examples: the key of 'C major' has 'A minor' as relative key"@en ,
+    "Exemples: la clé de 'Do majeur' a 'La mineur' pour tonalité relative"@fr .
+
+###  http://data.doremus.org/ontology#U84_has_parallel_key
+mus:U84_has_parallel_key rdf:type owl:SymmetricObjectProperty ;
+    rdfs:domain mus:M4_Key ;
+    rdfs:range mus:M4_Key ;
+    rdfs:isDefinedBy <http://data.doremus.org/ontology#> ;
+    rdfs:label "U84 has parallel key"@en , "U84 a pour tonalité parallèle"@fr ;
+    rdfs:comment "Scope Note: This class allows to connect two M4 Keys that are in a parallel relationship, i.e. one is the parallel major or minor of the other one."@en ,
+    "Scope Note: Cette classe permet de connecter deux M4 Tonalité qui sont en une relation parallèle, c'est-à-dire l'une est la parallèle majeur ou mineur de l'autre."@fr ,
+    "Examples: the key of 'C major' has 'C minor' as parallel key"@en ,
+    "Exemples: la clé de 'Do majeur' a 'Do mineur' pour tonalité parallèle"@fr .
+
+###  http://data.doremus.org/ontology#U85_has_closely_related_key
+mus:U85_has_closely_related_key rdf:type owl:SymmetricObjectProperty ;
+    rdfs:domain mus:M4_Key ;
+    rdfs:range mus:M4_Key ;
+    rdfs:isDefinedBy <http://data.doremus.org/ontology#> ;
+    rdfs:label "U85 has closely related key"@en , "U85 a pour tonalité étroitement lié"@fr ;
+    rdfs:comment "Scope Note: This class allows to connect two M4 Keys that are mutually closely related from an harmonic point of view, e.g. they are commonly used destinations or transpositions in a modulation."@en ,
+    "Scope Note: Cette classe permet de connecter deux M4 Tonalité qui sont étroitement liées d'un point de vue harmonique, par ex. ces sont des destinations ou des transpositions fréquemment utilisées dans une modulation"@fr ,
```

Comment:
Line 1633: s/ces sont/ceux sont

Reply:
You may also want to add the 3 wikipedia pages linked from https://github.com/DOREMUS-ANR/knowledge-base/issues/38 in the rdfs:comment as see also examples

Reply:
Ok for line 1633.

What about `rdfs:seeAlso`?

Reply:
I'm not saying you should add a rdfs:seeAlso property, I'm saying that you can extend your rdfs:comment with more text, and the same way you provide a "scope note" and an example, you can just add "see also https://en.wikipedia.org/wiki/Relative_key" or https://en.wikipedia.org/wiki/Parallel_key or https://en.wikipedia.org/wiki/Closely_related_key